cmd/helm/testdata/helmhome/helm/plugins/exitwith/plugin.yaml

```diff
@@ -0,0 +1,4 @@
+name: exitwith
```

Comment:
This is a helm plugin for testing purpose, that exits immediately with the given arg as the exit code. So `helm exitwith 2` should exit with 2 with means:

```
$ helm extiwith 1
$ echo $?
1
$ helm exitwith 2
$ echo $?
2
```